### PR TITLE
Add checks around meta overrides. Fixes #20

### LINKED ISSUE #20: Blog Plugin kills all meta-Infos from all other pages
If the blog plugin is active, all meta information of all other pages of the shop will be overwritten with "empty".

### PR DESCRIPTION
**Pull request by runelaenen**
Fixes #20 
Adds checks to see if route is the blog detail route before changing stuff.

### CODE REVIEW

File: src/Resources/views/storefront/layout/meta.html.twig [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
     {{ parent() }}
 {% endblock %}
 
-{% block layout_head_meta_tags_description %}{% if entry.translated.metaDescription %}{{ entry.translated.metaDescription|striptags|trim }}{% else %}{{ entry.translated.teaser }}{% endif %}{% endblock %}
+{% block layout_head_meta_tags_description %}{% if activeRoute == 'sas.frontend.blog.detail' %}{% if entry.translated.metaDescription %}{{ entry.translated.metaDescription|striptags|trim }}{% else %}{{ entry.translated.teaser }}{% endif %}{% else %}{{ parent() }}{% endif %}{% endblock %}
 
 {% block layout_head_title %}
-    {% set metaTitle = entry.translated.metaTitle|striptags|trim %}
+    {% if activeRoute == 'sas.frontend.blog.detail' %}
+        {% set metaTitle = entry.translated.metaTitle|striptags|trim %}
```
Comment (ChristopherDosin) [inline src/Resources/views/storefront/layout/meta.html.twig:14]: I think we should also check if the `metaTitle` exist, if not the `entry.translated.title` should be used in this case.

File: src/Resources/views/storefront/layout/meta.html.twig [hunk 2]
```diff
@@ -1,20 +1,24 @@
 {% sw_extends '@Storefront/storefront/layout/meta.html.twig' %}
 
 {% block layout_head_meta_tags %}
-    {% set metaTitle = entry.translated.metaTitle|striptags|trim %}
+    {% if activeRoute == 'sas.frontend.blog.detail' %}
+        {% set metaTitle = entry.translated.metaTitle|striptags|trim %}
```
Comment (ChristopherDosin) [inline src/Resources/views/storefront/layout/meta.html.twig:5]: I think we should also check if the metaTitle exist, if not the entry.translated.title should be used in this case.

File: src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js [hunk 3]
```diff
@@ -152,6 +152,7 @@ Component.register('sas-blog-detail', {
                     },
                     image: SimpleImage,
                     delimiter: Delimiter,
+                    raw: RawTool,
                     table: {
                         class: Table,
                         inlineToolbar: true,
```
Comment (ChristopherDosin) [inline src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js:158]: If possible move this to your #21 PR.

Comment (runelaenen) [reply src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js:158]: Ah I thought I did but I must've miss-pushed or something :p

**Pull request merged**